Node/src/TeamsChatConnector.ts

```diff
@@ -110,6 +112,17 @@ export class TeamsChatConnector extends builder.ChatConnector {
   }
 
   /**
+  * This method is to resolve regex or LUIS Bot being affected by at mentions
+  * Set true would have incoming messages stripped off source Bot at mentions
+  * The default would be false
```

Comment:
We should follow the design like BotBuilder SDK, it doesn't introduce any middleware plugin yet. And another concern I think is the compatibility, if we use restify or expressjs, there is middleware API however, it doesn't mean any node server would have such mechanism. Current method could be not very fancy but will work for most http servers.